ts/util/table-builder.ts

```diff
@@ -239,31 +234,31 @@ export class TableBuilder<ParentType, ChildType> {
       switch (filter.type) {
         case "dropdown":
           // create all submenu options
-          const reload = () => this.reload();
+          const onRemove = () => this.reload();
```

Comment:
Renamed in the wrong place. Think of the different contexts:

* In `AcceptedFitler`, the function is just a callback. `AcceptedFilter` doesn't know what the function will do, so the name here should be "onRemove"
* Here, we know what the function is doing - reloading the table. It's being used when adding filters as well as removing, so "onRemove" is a misleading name here and "reload" makes sense